pom.xml

```diff
@@ -96,6 +96,11 @@
 
   <dependencyManagement>
   <dependencies>
+    <dependency>
+      <groupId>commons-logging</groupId>
+      <artifactId>commons-logging</artifactId>
+      <version>1.2</version>
+    </dependency>
```

Comment:
Not imported directly, but a transitive dependency of several modules that causes Enforcer errors, so moved from `stapler-core` to here.